### PR TITLE
Dynamic addition for touch

### PR DESCRIPTION
**Pull request by jahorton**
Implements a fix for issue #62, allowing touch devices to have KeymanWeb support for dynamically-added elements.

Also fixes up some recent documentation, and comes with a test case webpage sample.

### CODE REVIEW

File: web/source/keymanweb.js [hunk 1]
```diff
@@ -3347,44 +3433,10 @@
...
-        e.kmwInput=false;                
-        if((!c || c.indexOf('kmw-disabled') < 0) && !e.readOnly)
-        {
-          if(e.type == 'text' || e.type == 'search') 
-          {
-            e.disabled=true; e.kmwInput=true;       
-          } 
-        }
-      }      
-    }
```
Comment (jahorton) [inline web/source/keymanweb.js]: This block was refactored into keymanweb.isKMWInput, as seen above.

File: web/source/keymanweb.js [hunk 2]
```diff
@@ -3347,44 +3433,10 @@
...
 
-    // Initialize and protect input elements for touch-screen devices (but never for apps)
-    // NB: now set disabled=true rather than readonly, since readonly does not always 
-    // prevent element from getting focus, e.g. within a LABEL element.
-    // c.f. http://kreotekdev.wordpress.com/2007/11/08/disabled-vs-readonly-form-fields/ 
-    if(device.touchable)
-    { 
-      eTextArea=document.getElementsByTagName("textarea");
-      eInput=document.getElementsByTagName("input");
-      for(i=0; i<eTextArea.length; i++)
```
Comment (jahorton) [inline web/source/keymanweb.js]: This block was refactored into keymanweb.isKMWInput, as seen above.

**Comment by mcdurdin** [comment]
LGTM. This will need some testing with kmwembedded.js to make sure that the changes don't impact Keyman for iOS and Keyman for Android.

**Comment by mcdurdin** [comment]
I'll merge after we discuss the kmwembedded.js scenarios

**Pull request merged**